src/tokens-studio/spectrum2-non-colors/spectrum2/layout.component/desktop.json

```diff
@@ -5059,6 +5059,28 @@
       }
     }
   },
+  "divider-horizontal-minimum-width": {
+    "value": "200px",
+    "type": "sizing",
+    "$extensions": {
+      "spectrum-tokens": {
+        "uuid": "",
+        "name": "divider-horizontal-minimum-width",
+        "constant-token-duplicate": false
```

Comment:
Could this be set to `true` if they are the same value for desktop and mobile?

Reply:
https://github.com/adobe/spectrum-tokens-studio-data/pull/47#issuecomment-1915676370